src/Services/ChargesService.php

```diff
@@ -72,7 +71,7 @@ public function create(Charge $charge): array
     public function generateQrCode(string $chargeId): string
     {
         $uri = sprintf(
-            'https://api.openpix.com.br/openpix/charge/brcode/image/%s.png?size=1024',
```

Comment:
Essa api é deiferente das demais. {host}/openpix e {host}/api

Reply:
I do think having a BASE_URL = https://api.openpix.com.br/

to be more productive

Reply:
Probably I'll change it to a config/larapix.php file and add everything that needs there after implement the Laravel Service Providers. For now I think it will work!

Reply:
Also this endpoint will not be continued on the early version because we still need to validate how to implement it with @sibelius and @daniloab or how to improve it.

Still a little bit confusing but if will works on React implementation.